UI/Source/CollectionViews/CollectionViewModelDataSource.swift

```diff
@@ -1178,7 +1177,7 @@ extension CollectionViewModelDataSource: NSCollectionViewDataSource {
         let reuseId = "\(NSStringFromClass(viewType))-\(type(of: viewModel))"
```

Comment:
Maybe this reuseId should change to be generic since as far as NSCV is concerned we're just reusing `CollectionViewHostReusableView`

Reply:
Not sure what you mean by generic? If we use the provided View/ViewModel combo we maximize the chance of returning a host view with the same hosted View type. Though I might be misunderstanding how this works...

In the interim I've updated this line and one other to use the `reuseIdProvider` to generate the ids so the entire file is consistent.